ui/src/app/auth/auth-roles.ts

```diff
@@ -3,3 +3,13 @@ export const OSMT_ADMIN = "ROLE_Osmt_Admin"
 export const OSMT_CURATOR = "ROLE_Osmt_Curator"
```

Comment:
You might consider removing the "export" on both of these to prevent other code from using them.  The reference to OSMT_ADMIN could be replaced with a reference through ROLES_AUTHORITY.

Reply:
Done deal